config/routes.rb

```diff
@@ -7,5 +7,17 @@
 
   resources :people
 
-  root 'people#index'
+  # Auth
+
+  get '/auth/:provider/callback' => 'sessions#create'
+
+  resource :sessions, :only => [:create]
+
+  get "/login" => redirect("/auth/twitter"), as: :login
+
+  delete "/logout" => 'sessions#destroy', as: :logout
+
+  root 'site#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.